### ISSUE TITLE
@CreateDateColumn() is not working with mongodb

### ISSUE THREAD

**Issue by valimero** [typeorm/typeorm#2938]
**Issue type:**

[ ] question
[x] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[x ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[ ] `postgres`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[ x] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

I'm using Nestjs/typeorm.

When I save the entity, there is only the id in mongodb:

**Entity:**
```  typescript
@Entity()
@Exclude()
export class Membre {
	@ObjectIdColumn()
	_id: ObjectID;

	@Column()
	public email: string;

	@Expose()
	@Column()
	public username: string;

	@Column()
	public password: string;

	@Column()
	public zipcode: string;

	@CreateDateColumn()
	public create_at: Date;

	@UpdateDateColumn()
	public update_at: Date;
}
```

**Repository:**
``` typescript
@EntityRepository(Membre)
export class MembreRepository extends MongoRepository<Membre>
{
	public  async getNewEmptyMembre(): Promise<Membre> {
		return await this.save(new Membre());
	}
}

**Comment by victordidenko** [comment]
Maybe changing `CreateDateColumn`/`UpdateDateColumn` to `new Date()` logic, as I suggested in issue #2939, will fix this also. So, values in these columns will not depend on database implementation of default values.

**Comment by trieudh58** [comment]
any progress on this?

**Comment by rustamwin** [comment]
#3335

**Comment by pleerock** [comment]
This should be fixed in 0.2.10.

**Comment by DmitryOlkhovoi** [comment]
I have the same issue with 0.2.26
